.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+
+
+```
+### ✨ Features
+
+*
+
+### 🐛 Bug Fixes
+
+*
+
+### 🔧 Tech
+
+*
+```
```

Comment:
`ci` scope doesn't exist. And even if it existed, this commit should have been tagged `chore`.